extensions/sql-migration/src/wizard/assessmentDetailsPage/assessmentDetialsBody.ts

```diff
@@ -243,19 +240,20 @@ export class AssessmentDetailsBody {
 			await this._issueSummary.refreshAssessmentDetails(selectedIssue);
 		}));
 
-		return resultContainer;
+		return summaryContainer;
 	}
 
+	// function to get latest issues/warnings list for the selected instance/database.
 	public async refreshResults(): Promise<void> {
-		if (this._targetType === MigrationTargetType.SQLMI ||
-			this._targetType === MigrationTargetType.SQLDB) {
+		if (this._model?._targetType === MigrationTargetType.SQLMI ||
+			this._model?._targetType === MigrationTargetType.SQLDB) {
```

Comment:
This is not required for target Type VM?

Reply:
I have taken this as part of pre existing logic. What I understood is, we don't receive any issues in SQL Server for VM. If there are issues in MI and DB, we direct them to VM.

Reply:
We direct issues from MI and DB to VM. Why?

Reply:
This is something pre existing.
There are always 0 issues in case of VM.

We direct customers with these warnings to select VM if there are issues in DB or MI:

"Databases that are not ready for migration to Azure SQL Database can be migrated to SQL Server on Azure Virtual Machines. Alternatively, review assessment results for Azure SQL Managed Instance migration readiness."

 "Databases that are not ready for migration to Azure SQL Managed Instance can be migrated to SQL Server on Azure Virtual Machines. Alternatively, review assessment results for Azure SQL Database migration readiness."